layouts/partials/pager.html

```diff
@@ -1,7 +1,7 @@
-<ul class="list-unstyled d-flex justify-content-between align-items-center mb-0 pt-5 d-print-none">
```

Comment:
@jcabak - what the removal of `d-print-none` voluntary? By doing so you're reverting #484.